charts/gitops-runtime/values.yaml

```diff
@@ -688,9 +682,9 @@ gitops-operator:
     # -- defaults
     registry: quay.io
     repository: codefresh/codefresh-gitops-operator
-    tag: "293f24f"
+    tag: "6882325"
   env:
-    <<:
+    !!merge <<:
```

Comment:
what is this?

Reply:
i'm also wondering. this all came from the automation commit that bumped the operator image tag.